### DISCUSSION TITLE
Environment variable or command line flag to turn off Velopack

### DISCUSSION THREAD

**Discussion by JeremyStarTM** [ppy/osu#30908] [Ideas] [answered]
I've written my own osu!lazer update script a while ago and it works fine to this day and I'd like to keep using it. It would be nice if I would be able to turn off Velopack (especially in regards to what happened to Linux users the past few weeks), either using a environment variable or a command line flag passed to the AppImage file. Is this a possibility?

**Comment by SupDos** [comment, marked as answer]
I believe you can set `OSU_EXTERNAL_UPDATE_PROVIDER=1` to do that
